Google Spreadsheet/GoogleSpreadsheet.Activities/ReadRange.cs

```diff
@@ -94,14 +95,58 @@ public static int GetNumberOfColumnsFromRange(string range)
             var firstLetter = (int)rangeParts[0][0];
             var secondLetter = (int)rangeParts[1][0];
 
-            if (secondLetter < firstLetter)
+            /* if (secondLetter < firstLetter)
+             {
+                 throw new Exception("Invalid range specified.");
+             }*/
+
+
+            // Handle also columns after AA
+            Regex regexRange = new Regex(@"([A-Z]+)\d+");
+
+            Match match = regexRange.Match(rangeParts[0]);
+            string firstColName, secondColName;
+            int firstColNumber = 0, secondColNumber = 0;
+            if (match.Success)
+            {
+                firstColName = match.Groups[1].Value;
+                firstColNumber = letterToColumn(firstColName);
+            }
+            else
+            {
+                throw new Exception("Invalid range specified.");
```

Comment:
We should try to throw more specific exception types like ArgumentException or InvalidOperation